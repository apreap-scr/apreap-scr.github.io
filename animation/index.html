<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
    <title>Document</title>
    <link rel="stylesheet" href="libs/reset.css">
    <link rel="stylesheet" href="libs/bootstrap-4.1.3-dist/css/bootstrap.min.css">
    <link rel="stylesheet" href="libs/slick/slick.css">
    <link rel="stylesheet" href="libs/slick/slick-theme.css">
    <link rel="stylesheet" href="libs/nav-menu/css/pushy.css">
    <link rel="stylesheet" href="fonts/flaticon/flaticon.css">
    <link rel="stylesheet" href="main.css">

</head>
<body>
<!-- Pushy Menu -->
<nav class="pushy pushy-right" data-focus="#first-link">
    <div class="pushy-content">
        <ul>
            <li class="pushy-submenu">
                <button id="first-link">Submenu 1</button>
                <ul>
                    <li class="pushy-link"><a href="#">Item 1</a></li>
                    <li class="pushy-link"><a href="#">Item 2</a></li>
                    <li class="pushy-link"><a href="#">Item 3</a></li>
                </ul>
            </li>
            <li class="pushy-submenu">
                <button>Submenu 2</button>
                <ul>
                    <li class="pushy-link"><a href="#">Item 1</a></li>
                    <li class="pushy-link"><a href="#">Item 2</a></li>
                    <li class="pushy-link"><a href="#">Item 3</a></li>
                </ul>
            </li>
            <li class="pushy-submenu">
                <button>Submenu 3</button>
                <ul>
                    <li class="pushy-link"><a href="#">Item 1</a></li>
                    <li class="pushy-link"><a href="#">Item 2</a></li>
                    <li class="pushy-link"><a href="#">Item 3</a></li>
                </ul>
            </li>
            <li class="pushy-submenu">
                <button>Submenu 4</button>
                <ul>
                    <li class="pushy-link"><a href="#">Item 1</a></li>
                    <li class="pushy-link"><a href="#">Item 2</a></li>
                    <li class="pushy-link"><a href="#">Item 3</a></li>
                </ul>
            </li>
            <li class="pushy-link"><a href="#">Item 1</a></li>
            <li class="pushy-link"><a href="#">Item 2</a></li>
            <li class="pushy-link"><a href="#">Item 3</a></li>
            <li class="pushy-link"><a href="#">Item 4</a></li>
        </ul>
    </div>
</nav>
<!-- Site Overlay -->
<div class="site-overlay"></div>
<div id="container">
    <div class="Header">
        <!-- --------------< header animation >------------------------------- -->
        <div class="Header-slider" id="headmov">
            <section class="animation slider" dir="rtl">
                <div class="itm itm1" dir="ltr">
                    <div class="itm-v itm-v1">
                        <a class="itm-logo" href="#"><img src="img/logo.png"></a>
                        <div class="itm-title">Connecting Business and Influencers</div>
                    </div>
                    <div class="itm-v itm-v2">
                        <div class="itm-buttons">
                            <div class="itm-buttons-block">
                                <div class="wrap-itm-buttons-ar">
                                    <i class="itm-buttons-ar flaticon-arrow"></i>
                                </div>
                                <i class="itm-buttons-ok flaticon-check-mark"></i>
                            </div>
                            <div class="itm-buttons-block">
                                <i class="itm-buttons-cl flaticon-error"></i>
                            </div>
                        </div>
                        <div class="itm-anne">
                            <div class="itm-anne-title">Anne,</div>
                            <div class="itm-anne-text">
                                Has a hat shop and wants to<br>promote her business
                            </div>
                        </div>
                    </div>
                    <div class="itm-v itm-v3">
                        <div class="itm-buttons">
                            <div class="itm-buttons-block">
                                <div class="wrap-itm-buttons-ar">
                                    <i class="itm-buttons-ar flaticon-arrow"></i>
                                </div>
                                <i class="itm-buttons-ok flaticon-check-mark"></i>
                            </div>
                            <div class="itm-buttons-block">
                                <i class="itm-buttons-cl flaticon-error"></i>
                            </div>
                        </div>
                        <div class="itm-chris">
                            <div class="itm-chris-title">Chris,</div>
                            <div class="itm-chris-text">
                                50K followers and loves<br>Anne’s ha
                            </div>
                        </div>
                    </div>
                </div>
                <div class="itm itm2" dir="ltr">
                    <div class="itm-v itm-v1">
                        <a class="itm-logo" href="#"><img src="img/logo.png"></a>
                        <div class="itm-title">Connecting Business and Influencers</div>
                    </div>
                    <div class="itm-v itm-v2">
                        <div class="itm-buttons">
                            <div class="itm-buttons-block">
                                <div class="wrap-itm-buttons-ar">
                                    <i class="itm-buttons-ar flaticon-arrow"></i>
                                </div>
                                <i class="itm-buttons-ok flaticon-check-mark"></i>
                            </div>
                            <div class="itm-buttons-block">
                                <i class="itm-buttons-cl flaticon-error"></i>
                            </div>
                        </div>
                        <div class="itm-anne">
                            <div class="itm-anne-title">Anne,</div>
                            <div class="itm-anne-text">
                                Has a hat shop and wants to<br>promote her business
                            </div>
                        </div>
                    </div>
                    <div class="itm-v itm-v3">
                        <div class="itm-buttons">
                            <div class="itm-buttons-block">
                                <div class="wrap-itm-buttons-ar">
                                    <i class="itm-buttons-ar flaticon-arrow"></i>
                                </div>
                                <i class="itm-buttons-ok flaticon-check-mark"></i>
                            </div>
                            <div class="itm-buttons-block">
                                <i class="itm-buttons-cl flaticon-error"></i>
                            </div>
                        </div>
                        <div class="itm-chris">
                            <div class="itm-chris-title">Chris,</div>
                            <div class="itm-chris-text">
                                50K followers and loves<br>Anne’s ha
                            </div>
                        </div>
                    </div>
                </div>
            </section>
        </div>
        <!-- --------------< End header animation >------------------------------- -->
        <div class="Header-fon">
            <div class="container">
                <div class="row">
                    <div class="col-lg-12">
                        <button class="menu-btn"><img src="img/burger.png"></button>
                        <div class="follow_us">
                            <a href="#" class="wrap-i"><i class="flaticon-twitter-logo-silhouette"></i></a>
                            <a href="#" class="wrap-i"><i class="flaticon-linkedin-logo"></i></a>
                            <a href="#" class="wrap-i"><i class="flaticon-facebook-letter-logo"></i></a>
                            <span class="text">Follow us</span>
                        </div>
                        <a class="logo" href="#"><img src="img/layer_2.png"></a>
                    </div>
                </div>
            </div>
        </div>
    </div>
    <div class="Logistics">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <div class="g__getapp getapp-media1">
                        <div class="g__getapp__text">Get the app</div>
                        <i class="flaticon-android-logo"></i>
                        <i class="flaticon-apple-logo"></i>
                    </div>
                    <div class="item-1-default">
                        <div class="item item-1">
                            <img src="img/tel1header.png">
                        </div>
                        <div class="text">Once connected, alignon the<br>
                            <span class="bold">logistics</span>
                        </div>
                        <div class="text_both">
                            <div class="left">
                                <span class="bold">Easyly find</span> the<br>
                                best influencer<br>
                                for your <span class="color">brand</span>
                            </div>
                            <div class="right">
                                <span class="bold">Easyly find</span> the<br>
                                best influencer<br>
                                for your <span class="color">brand</span>
                            </div>
                        </div>
                    </div>
                    <div class="item-2-media">
                        <div class="img-w-l">
                            <div class="img-w-l-img">
                                <img src="img/lefttel1media.png">
                            </div>
                            <div class="img-w-l-text">
                                <div class="right">
                                    <span class="bold">Easyly find</span> the<br>
                                    best influencer<br>
                                    for your <span class="color">brand</span>
                                </div>
                            </div>
                        </div>
                        <div class="img-w-r">
                            <div class="img-w-l-text">
                                <div class="left">
                                    <span class="bold">Easyly find</span> the<br>
                                    best influencer<br>
                                    for your <span class="color">brand</span>
                                </div>
                            </div>
                            <div class="img-w-l-img">
                                <img src="img/tel2rightmedia.png">
                            </div>
                        </div>
                        <div class="text">Once connected, alignon the<br>
                            <span class="bold">logistics</span>
                        </div>
                    </div>
                    <div class="item">
                        <img src="img/tel2header.png">
                    </div>
                    <div class="g__getapp getapp-media2">
                        <div class="g__getapp__text">Get the app</div>
                        <i class="flaticon-android-logo"></i>
                        <i class="flaticon-apple-logo"></i>
                    </div>
                </div>
            </div>
        </div>
    </div>
    <div class="Snikpic">
        <div class="line-vertical"></div>
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <div class="title"><a href="#"><img src="img/logo.png"></a></div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item-top">
                        <img src="img/rotate2.png">
                    </div>
                </div>
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item-top">
                        <img src="img/rotate1.png">
                    </div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_6.png"></div>
                        <div class="text">
                        <span class="light">YOU REACH MORE AUDIENCE<br>
                        <span class="up">WITH </span></span><span class="bold">AUTHENTIC CONTENT</span>
                        </div>
                    </div>
                </div>
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_7.png"></div>
                        <div class="text">
                    <span class="light">YOU HAVE FREE ACCESS TO ALL<br>
                    <span class="up">REQUESTS </span></span><span class="bold">FROM BUSINESSES</span>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="line-hor">
            <div class="fon"></div>
            <div class="fon"></div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_9.png"></div>
                        <div class="text">
                        <span class="light">YOU PAY LESS THAN FOR<br>
                        <span class="up">Any </span></span><span class="bold">OTHER OPTION</span>
                        </div>
                    </div>
                </div>
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_10.png"></div>
                        <div class="text">
                    <span class="light">YOU SELECT THE MOST RELEVANT<br>
                    <span class="up">CAMPAIGN </span></span><span class="bold">FOR YOUR AUDIENCE</span>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="line-hor">
            <div class="fon"></div>
            <div class="fon"></div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_12.png"></div>
                        <div class="text">
                        <span class="light">YOU CHOOSE HOW MUCH YOU<br>
                        <span class="up">SPEND </span></span><span class="bold">ON THE CAMPAIGN</span>
                        </div>
                    </div>
                </div>
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_13.png"></div>
                        <div class="text">
                    <span class="light">YOU DECIDE WHAT PRICE YOU<br>
                    <span class="up">WANT </span></span><span class="bold">TO CHARGE</span>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="line-hor">
            <div class="fon"></div>
            <div class="fon"></div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/vector_smart_object_15.png"></div>
                        <div class="text">
                        <span class="light">YOU CAN CREATE A CAMPAIGN<br>
                        <span class="up">IN LESS </span></span><span class="bold">THAN 5 MINUTES</span>
                        </div>
                    </div>
                </div>
                <div class="col-6 col-sm-6 col-lg-6">
                    <div class="item">
                        <div class="wrap-img"><img src="img/layer_59.png"></div>
                        <div class="text">
                    <span class="light">YOU ARE YOUR OWN<br>
                    <span class="up"></span></span><span class="bold">AGENT</span>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="line-hor">
            <div class="fon"></div>
            <div class="fon"></div>
        </div>
    </div>
    <div class="Map">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <div class="text">3,5 millions influencers available<br>
                        on the platform
                    </div>
                    <div class="free_block">
                        <div class="free_block-text">Free access</div>
                    </div>
                    <div class="g__getapp">
                        <div class="g__getapp__text">Get the app</div>
                        <i class="flaticon-android-logo"></i>
                        <i class="flaticon-apple-logo"></i>
                    </div>
                </div>
            </div>
        </div>
        <div class="elips"></div>
    </div>
    <div class="slider-three sliders">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <div class="person">
                        <div class="wrapp-imgages">
                            <div class="border-img"><img src="img/ellipse_7_3.png"></div>
                            <div class="border-img center"><img class="center" src="img/ellipse_7.png"></div>
                            <div class="border-img"><img src="img/ellipse_7_2.png"></div>
                        </div>
                    </div>
                    <section class="lazy slider" data-sizes="50vw">
                        <div class="slider-three__sl-1">
                            <blockquote>
                                <p>&Prime; Snikpic helped us launch our new retaurent. We have seen a significant
                                    uptake in our sales in less than a week &Prime;</p>
                            </blockquote>
                        </div>
                        <div class="slider-three__sl-1">
                            <blockquote>
                                <p>&Prime; Snikpic helped us launch our new retaurent. We have seen a significant
                                    uptake in our sales in less than a week &Prime;</p>
                            </blockquote>
                        </div>
                        <div class="slider-three__sl-1">
                            <blockquote>
                                <p>&Prime; Snikpic helped us launch our new retaurent. We have seen a significant
                                    uptake in our sales in less than a week &Prime;</p>
                            </blockquote>
                        </div>
                    </section>
                </div>
            </div>
        </div>
    </div>
    <div class="Contact_us">
        <div class="container">
            <div class="row">
                <div class="col-lg-10 center-block">
                    <div class="title">Contact Us</div>
                    <form action="">
                        <input type="name" placeholder="Name"><br>
                        <input type="email" placeholder="Email"><br>
                        <input type="text" placeholder="Subject"><br>
                        <textarea placeholder="Your Massage"></textarea>
                        <input type="submit" value="SUBMIT">
                    </form>
                </div>
            </div>
        </div>
    </div>
    <div class="Footer">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <div class="title">Subscribe</div>
                    <form action="">
                        <input type="email" placeholder="Email"><br>
                        <input type="submit" value="To our NewsLettere">
                    </form>
                </div>
                <div class="col-6 col-sm-4 col-lg-2">
                    <a href="#" class="name">Download</a>
                    <a href="#" class="text">Download for iOS</a>
                    <a href="#" class="text">Download for Android</a>
                </div>
                <div class="col-6 col-sm-4 col-lg-2">
                    <a href="#" class="name">Community</a>
                    <a href="#" class="text">Community Guidelines</a>
                    <a href="#" class="text">Safety Tips</a>
                    <a href="#" class="text">Support</a>
                </div>
                <div class="col-6 col-sm-4 col-lg-2">
                    <a href="#" class="name">News</a>
                    <a href="#" class="text">Blog</a>
                    <a href="#" class="text">Tech Blog</a>
                    <a href="#" class="text">Press</a>
                </div>
                <div class="col-6 col-sm-4 col-lg-2">
                    <a href="#" class="name">Partners</a>
                    <a href="#" class="text">Hey! VINA</a>
                    <a href="#" class="text">Spotyfy</a>
                </div>
                <div class="col-6 col-sm-4 col-lg-2">
                    <a href="#" class="name">Legal</a>
                    <a href="#" class="text">Privacy</a>
                    <a href="#" class="text">Terms</a>
                </div>
                <div class="col-6 col-sm-4 col-lg-2">
                    <a href="#" class="name">Company</a>
                    <a href="#" class="text">Jobs</a>
                    <a href="#" class="text">Contact</a>
                    <a href="#" class="text">Security</a>
                </div>
                <div class="col-lg-12">
                    <div class="reserved">2018 Snikpic All rights reserved</div>
                </div>
            </div>
        </div>
    </div>
</div>

<script src="https://code.jquery.com/jquery-2.2.0.min.js" type="text/javascript"></script>
<!--<script src="libs/Easing.js"></script>-->

<script src="libs/slick/slick.min.js"></script>
<script src="libs/nav-menu/js/pushy.js"></script>
<script src="js/script.js"></script>

</body>
</html>