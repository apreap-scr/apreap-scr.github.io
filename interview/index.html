<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    />
    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
    <link
      rel="stylesheet"
      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
      crossorigin="anonymous"
    />
    <title>Document</title>
    <style>
      /* .container {
            margin: 50px auto;
            width: 80%;
        }

        .row {

        }

        .col {
            margin-top: 20px;
        }
      */
      img {
        margin-top: 20px;
        width: 100%;
      }
      /* @media screen and (max-width: 992px) {
        img {
          margin-top: 20px;
          width: 100%;
        }
      } */
    </style>
  </head>
  <body>
    <div class="container">
      <div class="row">
        <div class="col-md-12"><img src="img/u6459nnamed.jpg" /></div>
        <div class="col-md-12"><img src="img/unn55amed.jpg" /></div>
        <div class="col-md-12"><img src="img/030920163094.jpg" /></div>
        <div class="col-md-12"><img src="img/030920163096.jpg" /></div>
        <div class="col-md-12"><img src="img/140520162919.jpg" /></div>
        <div class="col-md-12"><img src="img/240820163056.jpg" /></div>
        <div class="col-md-12"><img src="img/281120163199.jpg" /></div>
        <div class="col-md-12"><img src="img/IMG_1256йй.jpg" /></div>
        <div class="col-md-12"><img src="img/20150912_125626.jpg" /></div>

        <div class="col-md-12"><img src="img/01092011387.jpg" /></div>
        <div class="col-md-12"><img src="img/261120163185.jpg" /></div>
        <div class="col-md-12"><img src="img/281120163222.jpg" /></div>
        <div class="col-md-12"><img src="img/collage7.jpg" /></div>
        <div class="col-md-12"><img src="img/DSCN1112.JPG" /></div>
        <div class="col-md-12"><img src="img/DSCN1160.JPG" /></div>
        <div class="col-md-12"><img src="img/DSCN2795..jpg" /></div>
        <!-- <div class="col-md-12"><img src="img/DSCN3364.jpg" /></div>
        <div class="col-md-12"><img src="img/DSCN3403.jpg" /></div> -->
        <div class="col-md-12"><img src="img/DSCN6676.jpg" /></div>
        <div class="col-md-12"><img src="img/DSCN6921.JPG" /></div>
        <div class="col-md-12"><img src="img/IMG_1307йй.jpg" /></div>
      </div>
    </div>
  </body>
</html>
