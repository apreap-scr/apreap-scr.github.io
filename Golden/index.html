<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <link rel="stylesheet" href="fonts/flaticon/flaticon.css">
    <link rel="stylesheet" href="main.css">


</head>
<body>

<div class="main-wrap">

    <div class="Header">
        <div class="container">
            <div class="row">
                <div class="col col-1 logo">
                    <a href="#"><img src="img/logo-treehouse.png"></a>
                </div>
                <div class="col col-2">
                    <a href="#" class="menu-button" id="menuButton">
                        <span class="burger-icon"></span>
                    </a>
                    <div class="header_menu" id="header_menu">
                        <ul>
                            <li><a href="">Home</a></li>
                            <li><a href="">Services</a></li>
                            <li><a href="">Portfolio</a></li>
                            <li><a href="">About</a></li>
                            <li><a href="">Contact</a></li>
                        </ul>
                    </div>
                </div>
                <div class="col col-3 col-text">
                    <div class="text1">Welcome To Our Studio!</div>
                    <div class="text2">Itâ€™s nice to Meet you</div>
                    <a class="more" href="#">Tell Me More</a>
                </div>
            </div>
        </div>
    </div>

    <div class="Services">
        <div class="container">
            <div class="row">
                <div class="col col-1">
                    <div class="title">Services</div>
                    <div class="title2">Proin iaculis purus consequat sem cure.</div>
                </div>
                <div class="col col-2 itm">
                    <div class="view"><img src="img/serv1.png"></div>
                    <div class="name">E-Commerce</div>
                    <div class="text">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.
                    </div>
                </div>
                <div class="col col-2 itm">
                    <div class="view"><img src="img/serv2.png"></div>
                    <div class="name">E-Commerce</div>
                    <div class="text">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.
                    </div>
                </div>
                <div class="col col-2 itm">
                    <div class="view"><img src="img/serv3.png"></div>
                    <div class="name">E-Commerce</div>
                    <div class="text">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.
                    </div>
                </div>
            </div>
        </div>
    </div>

    <div class="Portfolio">
        <div class="container">
            <div class="row">
                <div class="col col-1">
                    <div class="title">Our portfolio</div>
                    <div class="title2">Proin iaculis purus consequat sem cure.</div>
                </div>
                <div class="col-2">
                    <div class="block">
                        <div class="wrap-img">
                            <img class="img" src="img/container_10.png">
                        </div>
                        <div class="text1">Ebony & Ivory</div>
                        <div class="text2">Branding</div>
                    </div>
                </div>
                <div class="col-2">
                    <div class="block">
                        <div class="wrap-img">
                            <img class="img" src="img/container_9.png">
                        </div>
                        <div class="text1">Smart Stationary</div>
                        <div class="text2">Print Design</div>
                    </div>
                </div>
                <div class="col-2">
                    <div class="block">
                        <div class="wrap-img">
                            <img class="img" src="img/container_8.png">
                        </div>
                        <div class="text1">Clever Poster</div>
                        <div class="text2">Print Design</div>
                    </div>
                </div>
                <div class="col-2">
                    <div class="block">
                        <div class="wrap-img">
                            <img class="img" src="img/container_7.png">
                        </div>
                        <div class="text1">Vinyl Record</div>
                        <div class="text2">Product Mock-Up</div>
                    </div>
                </div>
                <div class="col-2">
                    <div class="block">
                        <div class="wrap-img">
                            <img class="img" src="img/container_6.png">
                        </div>
                        <div class="text1">Treehouse Template</div>
                        <div class="text2">Web Design</div>
                    </div>
                </div>
                <div class="col-2">
                    <div class="block">
                        <div class="wrap-img">
                            <img class="img" src="img/container_5.png">
                        </div>
                        <div class="text1">Burned Logo</div>
                        <div class="text2">Branding</div>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <div class="About_us">
        <div class="container">
            <div class="row">
                <div class="col">
                    <div class="title">About Us</div>
                    <div class="title2">Proin iaculis purus consequat sem cure.</div>
                    <div class="desctop">
                        <div class="block block-left">
                            <div class="cont left">
                                <div class="text1">July 2010
                                    Our Humble Beginnings
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.
                                </div>
                            </div>
                            <div class="wrap-img"><img class="img" src="img/container-1.png"></div>
                        </div>
                        <div class="block block-right">
                            <div class="wrap-img"><img class="img" src="img/container-2.png"></div>
                            <div class="cont right">
                                <div class="text1">January 2011
                                    Facing Startup Battles
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit aenean
                                    rhoncus posuere odio in tincidunt.
                                </div>
                            </div>
                        </div>
                        <div class="block block-left">
                            <div class="cont left">
                                <div class="text1">December 2012
                                    Enter The Dark Days
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni.
                                </div>
                            </div>
                            <div class="wrap-img"><img class="img" src="img/container-3.png"></div>
                        </div>
                        <div class="block block-right">
                            <div class="wrap-img"><img class="img" src="img/container-4.png"></div>
                            <div class="cont right">
                                <div class="text1">february 2014
                                    Our Triumph
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit aenean.
                                </div>
                            </div>
                        </div>
                        <div class="block block-right">
                            <div class="wrap-img wrap-bottom">
                                <div class="wrap-bottom-text">
                                    Our<br>
                                    Story<br>
                                    Continues<br>
                                    ...
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="adapt">
                        <div class="block">
                            <div class="cont">
                                <div class="wrap-img"><img class="img" src="img/container-1.png"></div>
                                <div class="text1">July 2010
                                    Our Humble Beginnings
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.
                                </div>
                            </div>
                        </div>
                        <div class="block">
                            <div class="wrap-img"><img class="img" src="img/container-2.png"></div>
                            <div class="cont">
                                <div class="text1">January 2011
                                    Facing Startup Battles
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit aenean
                                    rhoncus posuere odio in tincidunt.
                                </div>
                            </div>
                        </div>
                        <div class="block">
                            <div class="cont">
                                <div class="wrap-img"><img class="img" src="img/container-3.png"></div>
                                <div class="text1">December 2012
                                    Enter The Dark Days
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni.
                                </div>
                            </div>
                        </div>
                        <div class="block">
                            <div class="wrap-img"><img class="img" src="img/container-4.png"></div>
                            <div class="cont">
                                <div class="text1">february 2014
                                    Our Triumph
                                </div>
                                <div class="text2">Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit aenean.
                                </div>
                            </div>
                        </div>
                        <div class="block">
                            <div class="wrap-img wrap-bottom">
                                <div class="wrap-bottom-text">
                                    Our<br>
                                    Story<br>
                                    Continues<br>
                                    ...
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <div class="Amazing">
        <div class="container">
            <div class="row">
                <div class="col col-1">
                    <div class="title">our amazing team</div>
                    <div class="title2">Proin iaculis purus consequat sem cure.</div>
                </div>
                <div class="col col-2">
                    <div class="wrap-img"></div>
                    <div class="text1">Kimberly Thompson</div>
                    <div class="text2">Marketer</div>
                    <div class="wrap-soc">
                        <div class="soc-but"><i class="flaticon-twitter-logo-silhouette"></i></div>
                        <div class="soc-but"><i class="flaticon-facebook"></i></div>
                        <div class="soc-but"><i class="flaticon-pinterest-logo"></i></div>
                    </div>
                </div>
                <div class="col col-2">
                    <div class="wrap-img"></div>
                    <div class="text1">Kimberly Thompson</div>
                    <div class="text2">Marketer</div>
                    <div class="wrap-soc">
                        <div class="soc-but"><i class="flaticon-twitter-logo-silhouette"></i></div>
                        <div class="soc-but"><i class="flaticon-facebook"></i></div>
                        <div class="soc-but"><i class="flaticon-pinterest-logo"></i></div>
                    </div>
                </div>
                <div class="col col-2">
                    <div class="wrap-img"></div>
                    <div class="text1">Kimberly Thompson</div>
                    <div class="text2">Marketer</div>
                    <div class="wrap-soc">
                        <div class="soc-but"><i class="flaticon-twitter-logo-silhouette"></i></div>
                        <div class="soc-but"><i class="flaticon-facebook"></i></div>
                        <div class="soc-but"><i class="flaticon-pinterest-logo"></i></div>
                    </div>
                </div>
                <div class="col col-1">
                    <div class="text-bottom">Proin iaculis purus consequat sem cure digni ssim donec porttitora entum
                        suscipit<br>
                        aenean rhoncus posuere odio in tincidunt proin iaculis.
                    </div>
                </div>
            </div>
        </div>
    </div>

    <div class="brend">
        <div class="container">
            <div class="row">
                <div class="col col-1">
                    <div class="brend-itm"><img src="img/microlancer_copy.png"></div>
                </div>
                <div class="col col-1">
                    <div class="brend-itm"><img src="img/envato.png"></div>
                </div>
                <div class="col col-1">
                    <div class="brend-itm"><img src="img/tuts.png"></div>
                </div>
                <div class="col col-1">
                    <div class="brend-itm"><img src="img/wordpress.png"></div>
                </div>
            </div>
        </div>
    </div>

    <div class="Contact">
        <div class="container">
            <div class="row">
                <div class="col col-1">
                    <div class="title">Contact us</div>
                    <div class="title2">Proin iaculis purus consequat sem cure.</div>
                </div>
                <form>
                    <div class="col col-2">
                        <input type="name" placeholder="your Name *"><br>
                        <input type="email" placeholder="your e-mail *"><br>
                        <input type="text" placeholder="subject"><br>
                    </div>
                    <div class="col col-2">
                        <textarea placeholder="Your message *"></textarea>
                    </div>
                    <div class="col col-1 center">
                        <input type="submit" value="Send Message">
                    </div>
                </form>
            </div>
        </div>
    </div>

    <div class="Footer">
        <div class="container">
            <div class="row">
                <div class="col col-1">
                    <p class="copy">Â© Copyright 2014 FreebiesXpress.com</p>
                </div>
                <div class="col col-1">
                    <div class="wrap-soc">
                        <div class="soc-but"><i class="flaticon-twitter-logo-silhouette"></i></div>
                        <div class="soc-but"><i class="flaticon-facebook"></i></div>
                        <div class="soc-but"><i class="flaticon-pinterest-logo"></i></div>
                    </div>
                </div>
                <div class="col col-1">

                </div>
            </div>
        </div>
    </div>

</div>

<script src="libs/jquery.min.js"></script>
<script src="js/script.js"></script>

</body>
</html>